ehitchhiking-ui/src/app/shared/models/user.ts

```diff
@@ -2,20 +2,24 @@ import {Car} from './car';
 export class User {
 	id: string;
 	name: string;
+	rate: number;
```

Comment:
rate is not a required field for User class

Reply:
removed